.styleguidist/icons.md

```diff
@@ -0,0 +1,95 @@
+Suomi.fi icons
```

Comment:
It's great to have all the icons displayed here!

I was just thinking, we don't have any place (neither here or in the icons repo) where there would be clear instruction how to use the components. Like:

```
import { IconAlert } from 'suomifi-icons'

<IconAlert color="red"/>
```